File: crates/cartograph-parser/tests/fixtures/python/strings.py
# Fixture: string structure. Nothing here is evaluated.
BASE = "/api/v1"
plain = "hello world"
empty = ""
escaped = "line\nbreak"
triple = """multi
line"""
fstring = f"/orders/{order_id}"
nested_f = f"{BASE}/orders/{order_id}/items"
no_subs = f"just text"
joined = "pre" + BASE + "post"
numeric = 1 + 2

File: crates/cartograph-parser/tests/fixtures/strings.ts
// Fixture: string and template structure. Nothing here is "resolved".
const BASE = "/api/v1";
const plain = "hello world";
const empty = "";
const escaped = "line\nbreak";
const template = `${BASE}/orders/${orderId}`;
const noSubs = `just text`;
const joined = "pre" + BASE + "post";
const numeric = 1 + 2; // no string operand: not a concatenation fact
declare const orderId: string;
